gapic/samplegen/samplegen.py

```diff
@@ -927,11 +925,7 @@ def parse_handwritten_specs(sample_configs: Sequence[str]) -> Generator[Dict[str
                     raise types.InvalidConfig(
                         "Sample config is invalid", valid)
```

Comment:
Outside the scope of the review, but taking a second look, this isn't very helpful. Do you think it would be better to include the config or the filepath tothe config here?